R/pkg/R/SQLContext.R

```diff
@@ -744,6 +747,9 @@ read.df.default <- function(path = NULL, source = NULL, schema = NULL, ...) {
   if (is.null(source)) {
     source <- getDefaultSqlSource()
   }
+  if (source == "csv" && is.null(options[["nullValue"]])) {
```

Comment:
Possibly. I wonder if we should conservative - since data source API is extensible - perhaps a new source `nullValue` could cause an unexpected behavior change?